js/ascendex.js

```diff
@@ -1167,6 +1168,51 @@ module.exports = class ascendex extends Exchange {
         }, market);
     }
 
+    async fetchTradingFees (params = {}) {
+        await this.loadMarkets ();
+        await this.loadAccounts ();
+        const account = this.safeValue (this.accounts, 0, {});
+        const accountGroup = this.safeString (account, 'id');
+        const request = {
+            'account-group': accountGroup,
+        };
+        const response = await this.v1PrivateAccountGroupGetSpotFee (this.extend (request, params));
+        //
+        //      {
+        //         code: '0',
+        //         data: {
+        //           domain: 'spot',
+        //           userUID: 'U1479576458',
+        //           vipLevel: '0',
+        //           fees: [
+        //             { symbol: 'HT/USDT', fee: { taker: '0.001', maker: '0.001' } },
+        //             { symbol: 'LAMB/BTC', fee: { taker: '0.002', maker: '0.002' } },
+        //             { symbol: 'STOS/USDT', fee: { taker: '0.002', maker: '0.002' } },
+        //             ...
+        //           ]
+        //         }
+        //      }
+        //
+        const data = this.safeValue (response, 'data', {});
+        const fees = this.safeValue (data, 'fees', []);
+        const result = {
+            'info': response,
+        };
+        for (let i = 0; i < fees.length; i++) {
+            const fee = fees[i];
+            const marketId = this.safeString (fee, 'symbol');
+            const symbol = this.safeSymbol (marketId, undefined, '/');
+            const takerMaker = this.safeValue (fee, 'fee', {});
+            result[symbol] = {
+                'info': {},
```

Comment:
Let's not put `info` in the result in this case, and put `fee` under each `info` in each symbol instead.